.ci/pytorch/win-test-helpers/installation-helpers/install_magma.py

```diff
@@ -0,0 +1,69 @@
+import os
+import subprocess
+import sys
+import pathlib
+
+
+cuda_version = os.environ['CUDA_VERSION']
+tmp_win_dir = os.environ['TMP_DIR_WIN']
+build_type = os.environ['BUILD_TYPE']
+
+
+if cuda_version == "cpu":
+
+    subprocess.run('echo skip magma installation for cpu builds', shell=True)
+    sys.exit(0)
+
+
+# remove dot in cuda_version, fox example 11.1 to 111
+
+
+if os.environ['USE_CUDA'] != "1":
+
+    sys.exit(0)
+
+
+if '.' not in cuda_version:
+
+    subprocess.run('echo CUDA version ' + cuda_version +
+        'format isn\'t correct, which doesn\'t contain \'.\'', shell=True)
+
+    sys.exit(1)
+
+os.system(str(pathlib.Path(__file__).parent.resolve()) + '\\magma_vars.bat')
+cuda_suffix = 'cuda' + cuda_version.replace('.', '')
+
+if cuda_suffix == '':
```

Comment:
I don't think `cuda_suffix` could ever be equal to empty string. It's adding in 'cuda' always. Is this a bug?